packages/node/src/behavior/system/network/ServerNetworkRuntime.ts

```diff
@@ -74,7 +74,7 @@ export class ServerNetworkRuntime extends NetworkRuntime {
             const port = this.owner.state.network.operationalPort;
             const options = this.owner.state.commissioning.mdns;
             const crypto = this.owner.env.get(Crypto);
-            const { server } = this.owner.env.get(MdnsService);
+            const { server } = this.owner.env.get(MdnsService, this.owner);
```

Comment:
Hmm with idea r.e. "asDependent" you could have the "services" be the thing you request services from, then you don't need to figure out what to use as the key